invoke_test.go

```diff
@@ -19,6 +19,19 @@ var (
 	}
 )
 
+func TestInvokeWithContextTimeout(t *testing.T) {
```

Comment:
Actually... if no timeout is specified to invoke then a timeout of 0 is assumed because of the nil value.
We can add a condition that ignores timeouts if 0 is passed, but the generator will always pass a default value > 0